src/widgets/BrandBlock/ui/BrandBlock.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { AppDispatch } from '@/app/providers/StoreProvider/config/store'
+import BrandCard from '@/entities/BrandCard'
+import HeadingBlock from '@/entities/HeadingBlock'
+import LinkButton from '@/entities/LinkButton'
+import { Routes } from '@/shared/config/routerConfig/routes'
+import { TEXT_ALL_BRANDS, TEXT_OUR_BRANDS } from '@/shared/constants/constants'
+import { useResize } from '@/shared/libs/hooks/useResize'
+import Scroll from '@/shared/ui/Scroll/Scroll'
+
+import { brandSelector } from '../selectors/selectors'
+import { fetchBrands } from '../slice/brandSlice'
+import { Brand } from '../types/types'
```

Comment:
типы лучше импортировать вот так:
```suggestion
import type { Brand } from '../types/types'
```